src/adapters/PeopleAdapter.js

```diff
@@ -6,11 +6,26 @@ import {throwError} from 'rxjs';
  * adhering to the exact parameters and structure of the returned objects
  */
 export default class PeopleAdapter {
+  /**
+   * A Person object with details about the person
+   * @typedef {object} PersonObject
+   * @param {string} id The person identifier
+   * @param {Array.<String>} emails An array of emails for the person
+   * @param {string} displayName The name to be displayed for the person
+   * @param {string} firstName The first name of the person
+   * @param {string} lastName The last name of the person
+   * @param {string} nickName The short name for the person
+   * @param {string} avatar The full url to the person's avatar
+   * @param {string} orgId The id of the organization the person belongs to
+   * @param {string} status The presence status of the user.
+   *                        One of: '', 'active', 'bot', 'call', 'dnd', 'group', 'inactive', 'meeting', 'ooo', 'presenting', 'self', 'typing'
+   */
+
   /**
    * Returns an observable that emits person data
    *
    * @param {String} id - ID of person to get
-   * @returns {Observable<PersonObject>}
+   * @returns {Observable.<PersonObject>}
```

Comment:
Can you also rename the class?
```suggestion
   * @returns {Observable.<Person>}
```